src/client/CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ CHANGELOG
 CASSH Client
 -----
 
+1.6.2
+-----
+
+2019/05/23
+
+### Bug Fixes
+
+  - fix "Error: No realname option given." : LDAP can be disable/enable
```

Comment:
I don't really understand the point for enabling / disabling LDAP.

I mean, if the user doesn't want to use LDAP or "deactivate" it, he would comment the lines in the config file wouldn't he ? So what is the need for a boolean option ?